src/modules/Massmailer/html_admin/mod_massmailer_index.html.twig

```diff
@@ -145,15 +145,20 @@
                             <div class="mb-3 row">
                                 <label class="col-md-3 col-form-label" for="test_client_id">{{ 'Client to send test email'|trans }}</label>
                                 <div class="col-md-6">
-                                    <select class="form-control autocomplete-selector"
-                                            placeholder="{{ 'Start typing the client name or ID'|trans }}"
-                                            id="test_client_id"
-                                            name="test_client_id"
-                                            data-resturl="admin/client/get_pairs"
-                                            data-csrf="{{ CSRFToken }}">
-                                        {% set test_client = admin.client_get({ 'id': params.test_client_id }) %}
-                                        <option value="{{ params.test_client_id }}" selected>{{ test_client.first_name }} {{ test_client.last_name }}</option>
-                                    </select>
+                                    {% if not request.test_client_id %}
+                                        <select class="form-control autocomplete-selector"
+                                                placeholder="{{ 'Start typing the client name or ID'|trans }}"
+                                                id="test_client_id"
+                                                name="test_client_id"
+                                                data-resturl="admin/client/get_pairs"
+                                                data-csrf="{{ CSRFToken }}">
+
```

Comment:
Blank space? 😛

Reply:
I have no idea what you're talking about 👀 
![image](https://user-images.githubusercontent.com/17304943/234377396-da8d27ec-140d-4f90-bb07-e27b3a0ee078.png)